substrate/frame/revive/src/vm/evm.rs

```diff
@@ -200,3 +200,24 @@ impl InputsTr for EVMInputs {
 		primitives::U256::ZERO
 	}
 }
+
+/// Blanket conversion trait between `sp_core::U256` and `revm::primitives::U256`
+trait U256Converter {
```

Comment:
Maybe vm/evm/instructions/utilty.rs would be a better place for this?

Reply:
leaving it here to avoid conflict for now we can revisit later